tasks/task_ont_medaka.wdl

```diff
@@ -62,6 +62,7 @@ task read_filtering {
 task consensus {
   input {
     String samplename
+    String? organism
```

Comment:
`organism` tag needs to be defined for TheiaCoV_Clearlabs workflow

Reply:
@sage-wright @kevinlibuit thanks!